config/routes.rb

```diff
@@ -868,6 +868,24 @@
     get "ensure_connection", controller: "oauth_clients", action: :ensure_connection, as: "oauth_clients_ensure_connection"
   end
 
+  namespace :scim_v2 do
```

Comment:
:yellow_circle: I realized that I just kinda accepted this whole namespace and naming of the scope so far, but now that I noticed I have to ask: Why do we call everything SCIM **v2**?

It's our first implementation of SCIM and I can't find a lot of official reference of the standard being called "version 2" in any way (some of the URNs contain an innocent `2.0` inside of them). That's setting it apart from OAuth 2.0, where the version number is even part of the name.

**tl;dr:** Is anything speaking against using `scim` as a name for our scopes and also as a namespace for our APIs?

Reply:
I believe it is due to protocol version we implement. And it is SCIM version 2.